codalab/lib/bundle_cli.py

```diff
@@ -2856,6 +2858,40 @@ def do_write_command(self, args):
         )
         print(target.bundle_uuid, file=self.stdout)
 
+    @Commands.command(
+        'open',
+        aliases=('o',),
+        help='Open a bundle detail page in a local web browser.',
```

Comment:
Does this actually open multiple bundles that match the bundle spec?

Reply:
yes, or you can specify multiple specs (e.g., `cl open ^1 ^2`).